packages/ndla-ui/src/locale/messages-nn.ts

```diff
@@ -955,15 +968,35 @@ const messages = {
     close: 'Lukk faktaboks',
   },
   myNdla: {
-    resources: 'Ressursar',
-    folders: 'Mapper',
+    resources: '{{count}} Ressurs',
+    resources_plural: '{{count}} Ressursar',
+    folders: '{{count}} Mappe',
+    folders_plural: '{{count}} Mappar',
```

Comment:
```suggestion
    resources: '{{count}} ressurs',
    resources_plural: '{{count}} ressursar',
    folders: '{{count}} mappe',
    folders_plural: '{{count}} mapper',
```
Er ikkje alltid automatisk a-ending i nynorsk.